content/rs/references/cli-utilities/rladmin/failover.md

```diff
@@ -0,0 +1,52 @@
+---
+Title: rladmin failover
+linkTitle: failover
+description: Fail over primary shards of a database to their replicas
+weight: $weight
+alwaysopen: false
+toc: "true"
+headerRange: "[1-2]"
+categories: ["RS"]
+aliases:
+---
+
+`rladmin failover` fails over one or more primary (also known as master) shards of a database, promoting their respective replicas to primary shards.
+
+``` sh
+rladmin failover
+        [db { db:<id> | name }]
+        shard <id1 ... idN>
+        [immediate]
+```
+
+### Parameters
+
+| Parameter | Type/Value                     | Description                                                                                   |
+|-----------|--------------------------------|-----------------------------------------------------------------------------------------------|
+| db        | db:\<id\><br /> name           | Fail over shards for the specified database                                                   |
+| shard     | one or more primary shard IDs  | Primary shard or shards to fail over                                                          |
+| immediate |                                | Perform failover without verifying the replica shards are in full sync with the master shards |
+
+### Returns
+
+`Finished successfully` if failover completed, `ERROR` otherwise.
```

Comment:
Should this be a full sentence? It reads a bit awkward this way.
(I'm no expert in command line reference pages, so disregard if this isn't right)

Reply:
Just fixed that. Thanks!